### PR TITLE
added dark theme

### PR DESCRIPTION
**Pull request by uditpadhan98**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
   <link rel="shortcut icon" href="favicon.ico">
   <link rel="apple-touch-icon" href="app-icon.png"/>
   <link rel="stylesheet" href="style.css">
-  <script src="index.js"></script>
+  <!-- <script src="index.js"></script> -->
```
Comment (lgarron) [inline index.html:12]: This should not move.

File: index.html [hunk 2]
```diff
@@ -20,6 +20,7 @@
 </noscript>
 
 <div class="before"></div>
+<img id="dark-theme" src="dark-theme.png" alt="">
```
Comment (lgarron) [inline index.html:23]: I have some concerns:

1. This is a fairly prominent location in the page, for a feature that does not affect testing functionality. This page needs to be easy to use for testing across a wide variety of devices, and "real estate" at the top of the screen should be considered expensive. I would advocate that we rely entirely on `prefers-color-scheme`, or find a way to tuck this e.g. in a corner.
2. Could you clarify where this image is from, and in particular whether it has a compatible license? Is there an SVG version, and version, and could we use SVG for a potential drop-shadow?
3. The alt tag should specify that this addresses dark mode.
4. Also for accessibility, I think this icon should have an associated tag.
5. The icon (or an associated UI element) should indicate whether dark mode is currently enabled, in an accessible way.
6. Does the icon work equally well in dark mode? I would imagine the drop shadows for that icon aren't quite designed for that.

File: style.css [hunk 3]
```diff
@@ -180,3 +180,14 @@ a#toggle:hover {
...
+/* for dark theme */
+#dark-theme{
+  height: 30px;
+  width: 30px;
+}
+
+.dark{
+  background-color: rgb(16, 39, 39);
+  color: aliceblue;
+}
```
Comment (lgarron) [inline style.css:193]: Please a newline at the end  of the file.

File: index.js [hunk 4]
```diff
@@ -658,3 +658,11 @@ window.addEventListener("load", function() {
   }
 
 });
+
+//for dark theme and light theme
+
+var icon=document.getElementById("dark-theme");
+icon.onclick=function(){
+    document.body.classList.toggle("dark");
+    document.querySelector("#toggle").style.backgroundColor="#F1F1F1";
```
Comment (lgarron) [inline index.js:667]: Ah, I missed this initially. I think this would just turn into a bright border for the image?

File: index.js [hunk 5]
```diff
@@ -658,3 +658,11 @@ window.addEventListener("load", function() {
   }
 
 });
+
+//for dark theme and light theme
+
+var icon=document.getElementById("dark-theme");
```
Comment (lgarron) [inline index.js:664]: Please match the formatting of the rest of the file (in particular, spacing around `=` and certain other syntactic symbols).

File: index.js [hunk 6]
```diff
@@ -658,3 +658,11 @@ window.addEventListener("load", function() {
   }
 
 });
+
+//for dark theme and light theme
+
```
Comment (lgarron) [inline index.js:663]:
```suggestion
```

Comment (uditpadhan98) [reply index.js:663]: Thanks for your review , I will try to make the necessary changes.

**Pull request closed without merging**